src/phases/replication/startup/StartupPhasesTemplate.yml

```diff
@@ -0,0 +1,234 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: |
+  Common definitions to support the workloads in replication/startup.
+
+NumPhases: &max_phase 13
+
+Document: &Doc
+  t: {^RandomInt: {min: 0, max: 10}}
+  w: {^RandomInt: {distribution: geometric, p: 0.1}}
+  x: 0
+  y: {^RandomInt: {min: 0, max: 1000}}
+  z: {^RandomInt: {min: 0, max: 2147483647}}  # This is for max int for low probability of conflicts
+  int0: &int {^RandomInt: {distribution: binomial, t: 200, p: 0.5}}
+  int1: *int
+  int2: *int
+  int3: *int
+  int4: *int
+  int5: *int
+  int6: *int
+  int7: *int
+  int8: *int
+  int9: *int
+  intArray:
+  - 1000
+  - 2000
+  - 3000
+  - 4000
+  - 5000
+  - 6000
+  - 7000
+  - 8000
+  - 9000
+  string0: &string {^RandomString: {length: {^RandomInt: {min: 5, max: 15}}}}
+  string1: *string
+  string2: *string
+  string3: *string
+  string4: *string
+  string5: *string
+  string6: *string
+  string7: *string
+  string8: *string
+  string9: *string
+  stringShort: {^RandomString: {length: 1}}
+  stringLong: {^RandomString: {length: 100}}
+  padding: {^FastRandomString: {length: {^Parameter: {Name: "approxDocumentSize", Default: 100}}}}
+
+IndexesForLoader: &IndexesForLoader  # 10 indexes
+- keys: {t: 1, w: 1}
+- keys: {y: 1}
+- keys: {t: 1, stringShort: 1}
+- keys: {string0: 1}
+- keys: {stringLong: 1}
+- keys: {stringShort: 1, y: 1}
+- keys: {int0: 1, int1: 1, int2: 1}
+- keys: {int3: 1}
+- keys: {int4: 1}
+- keys: {int5: 1, int6: 1}
+
+IndexesForCreateCmd: &IndexesForCreateCmd  # 10 indexes
+- key: {t: 1, w: 1}
+  name: "index_t_w"
+- key: {y: 1}
+  name: "index_y"
+- key: {t: 1, stringShort: 1}
+  name: "index_t_stringShort"
+- key: {string0: 1}
+  name: "index_string0"
+- key: {stringLong: 1}
+  name: "index_stringLong"
+- key: {stringShort: 1, y: 1}
+  name: "index_stringShort_y"
+- key: {int0: 1, int1: 1, int2: 1}
+  name: "index_int0_int1_int2"
+- key: {int3: 1}
+  name: "index_int3"
+- key: {int4: 1}
+  name: "index_int4"
+- key: {int5: 1, int6: 1}
+  name: "index_int5_int6"
+
+InsertDataTemplate:
+  Name: InsertData
+  Type: Loader
+  Threads: 100
+  Phases:
+    OnlyActiveInPhases:
+      Active: {^Parameter: {Name: "active", Default: [0]}}
+      NopInPhasesUpTo: {^Parameter: {Name: "nopInPhasesUpTo", Default: *max_phase}}
+      PhaseConfig:
+        Threads: 100
+        Repeat: 1
+        Database: {^Parameter: {Name: "database", Default: test}}
+        CollectionCount: {^Parameter: {Name: "collectionCount", Default: 1}}
+        DocumentCount: {^Parameter: {Name: "documentCount", Default: 1e6}}
+        BatchSize: {^Parameter: {Name: "batchSize", Default: 10000}}
+        Document: *Doc
+        Indexes: {^Parameter: {Name: "indexes", Default: *IndexesForLoader}}
+
+CrudOperationsTemplate:
```

Comment:
I'm assuming you don't have a metrics name as these operations are not important?

Reply:
Yes